tests/drivers/rtc/rtc_api/doc/rtc_test.rst

```diff
@@ -0,0 +1,31 @@
+.. _rtc_api_test:
+
+:Author: Bjarki Arge Andreasen
```

Comment:
Non-blocking comment - I don't see authors tagged like this anywhere else in the documentation.  @carlescufi what are your thoughts on this?

Reply:
I have removed it :)

The file itself has been merged with the RTC documentation rtc.rst, since I don't think the tests folder is crawled through when creating documentation.

I initially added it to mimic the Apache + Copyright we have in source code, but as pointed out I don't see it elsewhere